CONTRIBUTING.md

```diff
@@ -21,3 +21,25 @@ The build tools for v140 (Platform Toolset = 'v140') cannot be found. To build u
 Retarget the solution to v141:
 1. Right click the ReactNative solution and click "Retarget Solution"
 2. Make sure the Platform Toolset has Upgrade to v141 selected, and click "OK."
+
+
+## Using RNTester
+RNTester is a React Native Windows App that demonstrates the implemented views and modules of React Native Windows.
+
+You can use it to test your changes to react-native-windows by making sure your changes haven't broken the views and modules.
```

Comment:
Inconsistent use of `react-native-windows` and `React Native Windows` in this section.